docs/source/starthere/migration-guide.rst

```diff
@@ -0,0 +1,70 @@
+Migration guide to use lightning 2.0
+============
+
+.. # define a hard line break for html
+.. |br| raw:: html
+
+    <br />
+
+.. _dummy_header:
+
+* Replace ``trainer.strategy=null`` with ``trainer.strategy=auto`` as 
+  `lightning 2.0 doesn't have None strategy <https://lightning.ai/docs/pytorch/stable/common/trainer.html#:~:text=strategy%20(Union%5Bstr%2C%20Strategy%5D)%20%E2%80%93%20Supports%20different%20training%20strategies%20with%20aliases%20as%20well%20custom%20strategies.%20Default%3A%20%22auto%22.>`_.
+..
+* Remove ``resume_from_checkpoint`` if being used as a trainer flag and pass the path to 
+  `Trainer.fit(ckpt_path="...") method <https://lightning.ai/docs/pytorch/stable/upgrade/from_1_9.html#:~:text=used%20Trainer%E2%80%99s%20flag%20resume_from_checkpoint>`_.
+..
+* Set ``trainer.strategy = "ddp_find_unused_parameters_true"`` if there are unused parameters in your model as lightning 2.0 has find_unused_parameters as False by default. 
```

Comment:
Is this the right syntax?

Reply:
Yes it is: [we use it here](https://github.com/NVIDIA/NeMo/pull/6433/files#:~:text=Resolve%20conversation-,cfg.trainer.strategy%20%3D%20%22ddp_find_unused_parameters_true%22,-logging.info)

Reply:
Also used in the lightning's PR: [strategy="ddp_find_unused_parameters_true"](https://github.com/Lightning-AI/lightning/pull/16611/files#:~:text=%60strategy%3D%22ddp_find_unused_parameters_true%22%60)